jfrog-cli/artifactory/utils/buildinfoproperties_test.go

```diff
@@ -37,19 +57,30 @@ func testCreateDefaultPropertiesFile(buildType BuildType, t *testing.T) {
 		}
 	}
 
+	for key, value := range propertiesFileConfig {
```

Comment:
Question - why do we need both the * propertiesFileConfig* and *expectedConfig* maps? Aren't they always identical?

Reply:
Not exactly. The expectedConfig is a Viper not a map. After creating the viper and populating the viper information we adding the propertiesFileConfig key and value to the viper. We can change this to:

	expectedConfig.Set("artifactory.proxy.host", host)
	expectedConfig.Set("artifactory.proxy.port", port)

Reply:
I've changed it per our conversation.